src/test/java/com/myhome/services/HouseMemberDocumentServiceTestAdditional.java

```diff
@@ -0,0 +1,179 @@
+package com.myhome.services;
+
+import com.myhome.domain.HouseMember;
+import com.myhome.domain.HouseMemberDocument;
+import com.myhome.repositories.HouseMemberDocumentRepository;
+import com.myhome.repositories.HouseMemberRepository;
+import com.myhome.services.springdatajpa.HouseMemberDocumentSDJpaService;
+import org.junit.Ignore;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.mock.web.MockMultipartFile;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@SpringBootTest
+public class HouseMemberDocumentServiceTestAdditional {
+
+    private static final String MEMBER_ID = "test-member-id";
+    private static final String MEMBER_NAME = "test-member-name";
+    private static final HouseMemberDocument MEMBER_DOCUMENT = new HouseMemberDocument("test-file-name", new byte[0]);
+
+    @Value("${files.compressionBorderSizeKBytes}")
+    private int compressionBorderSizeKBytes;
+
+    @MockBean
+    private HouseMemberRepository houseMemberRepository;
+
+    @MockBean
+    private HouseMemberDocumentRepository houseMemberDocumentRepository;
+    
+    @Autowired
+    private HouseMemberDocumentSDJpaService houseMemberDocumentService;
+
+    @BeforeEach
+    private void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    private byte[] getImageAsByteArray(int height, int width) throws IOException {
```

Comment:
Could you move this method to the end of this class?
Test classes should have tests and then 'helpers' methods